### DISCUSSION TITLE
Extend Canvas to use it as a Backend

### DISCUSSION THREAD

**Discussion by Adioz01** [austintoddj/canvas#1002] [General]
Great App! I really enjoy it.

Short question: Is it possible to extend  Canvas to use it as a backend? 

I'm thinking about adding custom sections that i would need.

Best,
Ray

**Comment by austintoddj** [comment]
Glad you like it! Are you wanting to see it with more of a CMS functionality?

**Comment by Adioz01** [comment]
Not really more functionality like a CMS but in my case i would have liked to add spatie media library to your tool.

:)

**Reply by austintoddj** [reply]
Ah I see. I don't have plans at this time to extend it like you're describing.

**Comment by austintoddj** [comment]
Closing as obsolete with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).
